core-java-modules/core-java-io-5/src/test/java/com/baeldung/usinggzipInputstream/ReadingGZIPUsingGZIPInputStreamUnitTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.usinggzipInputstream;
+
+import org.junit.jupiter.api.Test;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.zip.GZIPInputStream;
+
+import static org.junit.Assert.assertEquals;
+
+public class ReadingGZIPUsingGZIPInputStreamUnitTest {

```

Comment:
JUnit 5 test classes should not be `public` - they can be package private - i.e. no modifier.